docs/_docs/reference/constraint.md

```diff
@@ -188,3 +188,104 @@ trait RefinedTypeOps[A, C, T]:
   def option(value: A): Option[T] =
     Option.when(rtc.test(value))(value.asInstanceOf[T])
 ```
+
+# Pre-defined constraints
```

Comment:
This section (and children) should have an extra `#` as a `h1` (single `#`) is used for the title.

Reply:
Fixed in the next commit